build/sections/founder.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+    <div class="pt-36">
+        <div class="flex flex-col w-full justify-center md:justify-start text-white font-raleway md:w-[80%] py-5">
+            <h2 class="font-bold text-2xl md:text-4xl font-raleway">Die <span class="text-primary-green">Founder</span> und ihre Vision von Ojemba</h2>
+            <div class="py-10 text-base font-montserrat font-medium w-full md:w-[90%] lg:w-[75%] 2xl:w-[60%]">
+                <p class="pb-3">
+                    Wir werden auf dem afrikanischen Kontinent Wohlstand schaffen, 
+                    indem wir respektvolle gegenseitige Geschäftsbeziehungen 
+                    zwischen dem Norden und dem Süden des Planeten aufbauen. 
+                </p>
+                <p>
+                    Dies wollen wir erreichen, indem wir wir IT-Talente aus dem 
+                    Süden mit IT-Möglichkeiten aus dem Norden zusammenbringen, 
+                    indem wir junge afrikanische Software-Ingenieure ausbilden 
+                    und ihnen anspruchsvolle Jobs für Unternehmen aus dem Norden 
+                    und Süden anbieten.
+                </p>
+            </div>
+        </div>
+    </div>
+    <div class="flex justify-center">
+        <img src="Images/Objekt.webp" alt="people seated" class="w-[90%] lg:w-[65%] 2xl:w-[70%] mx-auto" />
```

Comment:
- I have changed `w-[65%]` since its change isn't affecting the work. For `w-[70%] and w-[90%]` are not in tailwindcss. When I try changing them to match with the ones the tailwindcss it gets disorganised a bit

Reply:
You can follow as we discussed in the meeting.